### PR TITLE
[24.x] Bump version to 24.0rc1

### PR DESCRIPTION
**Pull request by fanquake**
Bump the version to 24.0rc1
Generate the man pages.
Generate the example bitcoin conf file.
Update the version number in bips.md.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -1,9 +1,9 @@
 AC_PREREQ([2.69])
-define(_CLIENT_VERSION_MAJOR, 23)
-define(_CLIENT_VERSION_MINOR, 99)
+define(_CLIENT_VERSION_MAJOR, 24)
+define(_CLIENT_VERSION_MINOR, 0)
 define(_CLIENT_VERSION_BUILD, 0)
 define(_CLIENT_VERSION_RC, 0)
```
[the remark this answers is not in the document]
Comment (fanquake) [reply configure.ac:5]: Thanks. I'll also need to regenerate them to pickup a missing option.

**Review by glozow — COMMENTED** [review]
in 4d789c07010fedd387e4a82333b561b7ed3276a6: The manpages generated when I ran gen-manpages.py includes `-sandbox` in bitcoin-qt and bitcoind (diff [here](https://github.com/glozow/bitcoin/commit/bdcfbef6ba768f1f7fe03c7626aa201bd7d1fba5))

**Comment by fanquake** [comment]
I've also added a commit to bump the version number in bips.md.

**Comment by achow101** [comment]
Should `sandbox` also be in the generated bitcoin.conf?

**Comment by fanquake** [comment]
> Should sandbox also be in the generated bitcoin.conf?

Added.

**Comment by achow101** [comment]
ACK ab4a32b8aff14da3f6d116b6a81ad0bb29397f1c

**Review by jarolrod — COMMENTED** [review]
post merge ack ab4a32b8aff14da3f6d116b6a81ad0bb29397f1c

**Pull request merged**